public/services/time-service.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wazuh app - Time and date functions
+ * Copyright (C) 2015-2019 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+export class TimeService {
+  /**
+   * Returns given date adding the timezone offset
+   * @param {string} date Date
+   */
+  offset(d) {
+    try {
+      const date = new Date(d);
+      const offset = new Date().getTimezoneOffset();
+      const offsetTime = new Date(date.getTime() - offset * 60000);
+      return offsetTime.toLocaleString('en-ZA').replace(',', '');
```

Comment:
Why `en-ZA`?

Reply:
BEcause according to this: http://www.localeplanet.com/icu/en-ZA/index.html, this country (South-Africa) uses the exact format we need (yyyy/MM/dd hh:mm:ss).